sounds.py

```diff
@@ -1,22 +1,48 @@
 from collections import namedtuple
+from dataclasses import dataclass
+from dotenv import load_dotenv
+from google.cloud import storage
 import speech_recognition as sr
-from pathlib import Path 
+from speech_recognition import AudioData
 import os
+from pathlib import Path 
 import re
```

Comment:
a little flake8 / PEP8 pass could help make these import sections a bit cleaner

Reply:
flake8 seems to be okay with this actually